.travis.yml

```diff
@@ -16,6 +16,9 @@ addons:
   apt:
     packages:
       - xsltproc
+      - libreoffice
+      - libtiff-tools
+      - ghostscript
```

Comment:
Is the packaging for the RPM getting updated with these? Can they be dependent on kazoo-application-fax (and whichever other apps might use them)?

Reply:
We could make this a dependency of kazoo-applications. Though we might want to exclude libreoffice from that list since i think we will make it default to disabled